nuxt.config.js

```diff
@@ -90,8 +96,9 @@ export default {
   },
 
   axios: {
+    proxyHeaders: false,
     credentials: false,
-    baseURL: 'https://beta.colmenaproject.es/admin/api/1.0/',
+    baseURL: 'http://localhost/colmena-manager/api/1.0/',
```

Comment:
@Borjarguez Please review the notation of API because I'm not sure about including 1.0 on the URL, maybe it should be something like "v1". Please confirm

Reply:
Maybe renaming the API's notation is not a bad idea. I'll make the change throughout the morning